### ISSUE TITLE
Implement limit checks.

### ISSUE THREAD

**Issue by pchote** [rockit-astro/mountd-talon#8]
Right now if you issue a move command in the limit gap it will time out and fail.  We should be checking against the HA / Dec limits in the talon config and eagerly failing.

**Comment by pchote** [comment]
Implemented in 43c5eb7fe7251aebbab537a141cb3e857f7fec6e.  Needs testing.

**Comment by pchote** [comment]
After homing the telescope from a clean start I get the following when trying to `tel park stow`:

```
Mar 09 11:38:45 sqftnuc1 env[3094]: failed to move to alt 0.616 az 0.405; ha 5.14300607127103 outside limit (-1.451028, 1.126845)
```

It looks like I need to wrap the calculated angle to the range `-Pi...Pi`.  Subtracting 2Pi from 5.143 gives -1.14, which is safely inside the limits.

**Comment by pchote** [comment]
This is because `self.__local_sidereal_time() - ra_radians` in `__ha_dec_from_ra_dec` doesn't account for the wraparound when the current -> requested RA goes through 24/00.

**Comment by pchote** [comment]
Fixed wraparound issue in 40bebbd393a1e536d663bf91ae3397507d76a404.
Still need to properly test coordinates outside limits.

**Comment by pchote** [comment]
Finally fixed properly in 2bb1780e36b3db40f40bcafca94ce22efdc21de8
